lib/open_project/translations/models/locales_updater.rb

```diff
@@ -0,0 +1,187 @@
+# todo those ought to be in the gemspec
+require 'tempfile'
+require 'crowdin-api'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.